src/xsar/sentinel1_xml_mappings.py

```diff
@@ -121,10 +122,19 @@ def list_poly_from_list_string_coords(str_coords_list):
     'annotation': {
         'atrack': (uniq_sorted, '/product/geolocationGrid/geolocationGridPointList/geolocationGridPoint/line'),
         'xtrack': (uniq_sorted, '/product/geolocationGrid/geolocationGridPointList/geolocationGridPoint/pixel'),
+        'atrack_grid': (int_array, '//product/geolocationGrid/geolocationGridPointList/geolocationGridPoint/line'),
+        'xtrack_grid': (int_array, '//product/geolocationGrid/geolocationGridPointList/geolocationGridPoint/pixel'),
```

Comment:
I think that an absolute xpath should be fine ('/product', not '//product')

Reply:
correction done!